packages/next/package.json

```diff
@@ -75,6 +75,7 @@
     "caniuse-lite": "^1.0.30001113",
     "chalk": "2.4.2",
     "chokidar": "3.4.2",
+    "critters": "0.0.6",
```

Comment:
Should we make this an optional peer dependency while it's experimental since the package is a couple megabytes? https://packagephobia.com/result?p=critters